kolibri/plugins/learn/assets/src/views/ExploreLibrariesPage/index.vue

```diff
@@ -211,8 +233,7 @@
         return this.createPinForUser(instance_id).then(response => {
           const id = response.id;
           this.usersPins = [...this.usersPins, { instance_id, id }];
-          this.moreDevices = this.unpinnedDevices;
-
+          this.moreDevices = 0;
```

Comment:
For purposes of clarity, why are we resetting this to `0`?

Reply:
Because, by default, if there are any pinned devices we don't display 'more devices' - we could attempt to preserve user intent if they have already pressed 'show more' but this was simpler to do for now.